libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -167,6 +179,25 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
       const { ConsoleTimelinePlugin } = await import('../../logic/operations/ConsoleTimelinePlugin');
       new ConsoleTimelinePlugin(terminal).apply(this.hooks);
     }
+
+    const showDependencyAnalysis: boolean = this._dependencyAnalysisParameter
+      ? this._dependencyAnalysisParameter.value
+      : false;
+    if (showDependencyAnalysis) {
+      const telemetryFolderName: string = path.join(this.rushConfiguration.commonTempFolder, 'telemetry');
+      const fullPath: string = path.join(telemetryFolderName, this._telemetryFileParameter?.value ?? '');
+      dependencyAnalysis(fullPath, terminal);
+      return;
+    }
+
+    const doSimulation: boolean = this._simulateParameter ? this._simulateParameter.value : false;
+    if (doSimulation) {
+      const telemetryFolderName: string = path.join(this.rushConfiguration.commonTempFolder, 'telemetry');
+      const fullPath: string = path.join(telemetryFolderName, this._telemetryFileParameter?.value ?? '');
+      simulateBuildTime(fullPath, terminal);
+      return;
+    }
```

Comment:
These would be better handled as a completely separate `rush analyze` or other action, since they don't depend on any of the context of the build itself and are purely post-build static analysis operations.

Reply:
Good call. I was able to add an Analyze action triggered via `rush analyze`